model/comments-model.js

```diff
@@ -1,4 +1,9 @@
 const db = require("../db/connection");
+const users = require("../db/data/test-data/users");
```

Comment:
While this solution for checking a valid user works right now, it will break if any new users are added to the DB at a later point